clubs/tests/test_views.py

```diff
@@ -21,7 +21,8 @@ def setUp(self):
             location='Somewhere',
             latitude=5,
             longitude=6,
-            status='approved'
+            status='approved',
+            denial='approved'
```

Comment:
We'd normally do that as the field being set to empty, unless the status field is in a certain condition. But this is normally handled in Django by forms and processing the input to the model. That's more complex than you want this example app to be, so hey whatevs.